sdks/java/core/src/main/java/org/apache/beam/sdk/testing/StreamingIT.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.beam.sdk.testing;
+
+/**
+ * Category tag that is used to validate tests which are expected running
```

Comment:
Category tag used to mark tests which execute using the Dataflow runner in streaming mode.

I would add deprecated to this annotation so we will be able to remove it since we don't want to have tests differentiated on streaming or not streaming since that is an implementation detail of the DataflowRunner.

In reality, we want tests to be tagged with Bounded and Unbounded to signify which they are testing.